src/utils/data.js

```diff
@@ -73,7 +74,8 @@ function transformDataPathToRoutePath(dataURL) {
 }
 
 export async function fetchDataForRouteEnter(to, from, next) {
-  const path = createDataPath(to.path);
+  const localePath = to.params.locale ? to.path : defaultLocale + to.path;
```

Comment:
I'm not sure this default scenario will work well, because it will break all existing content that hasn't yet been recompiled with a version of DocC that supports localized content.

For example, if I don't yet have a data source with a `en-US` subdirectory, existing URLs will fail to load.

Can we just fall back to not including a locale in the path if a specific locale has been requested? Otherwise, this will be a major breaking API change where this version of the renderer wouldn't work with any older content.

Essentially, I think I should be able to run the following and still be able to load the page at /documentation/docc:
```
env VUE_APP_DEV_SERVER_PROXY=https://mportiz08.github.io/swift-docc npm run serve
```

With this PR, it is a 404 right now since the URL doesn't yet include the locale parameter. I think we should try to make this backwards-compatible if at all possible.

Reply:
I totally agree. I created and centralised the `enablei18n` property into the `src/lang/index.js` file.

The idea is to have the i18n feature flag turned off for older doccarchives. Turning it on wouldn't add any value if they don't have localised content files anyways.